PCGen-Formula/code/src/java/pcgen/base/formula/visitor/StaticVisitor.java

```diff
@@ -118,8 +118,7 @@ public StaticVisitor(FunctionLibrary fl)
 	 * 
 	 * {@inheritDoc}
 	 */
-	@Override
-	public Object visit(SimpleNode node, Object data)
+	public Object visit(SimpleNode node)
```

Comment:
compiles for me though that might be some artifact of how I generated this cherry-pick (I only tested the "everything at once" variant).
Since data is always null do we still need this in the interface?